README.md

```diff
@@ -114,14 +114,15 @@ To test the changes on a mobile or virtual machine, you will need to open the so
 
 ### Testing in a consuming project
 
-- Run `yarn run build-consumer PROJECT_DIRECTORY`
+1. In your terminal, run `yarn run build-consumer PROJECT_DIRECTORY` for the polaris-react repo
 
 `PROJECT_DIRECTORY` is where the build will be copied, which must be a sibling of the `polaris-react` directory.
 
 ```sh
 # Example
 yarn run build-consumer polaris-styleguide
 ```
+2. In your terminal, open a second tab and run `dev server` for the polaris-styleguide repo
```

Comment:
We probably want to mention something about running `yarn run dev` instead of `dev server` (same thing, `dev server` is just a dev alias for `yarn run dev`) for non-Shopify people who do not have `dev`